mender-auth/cli/actions.cpp

```diff
@@ -80,6 +82,39 @@ error::Error DoBootstrap(shared_ptr<MenderKeyStore> keystore, const bool force)
 			return err;
 		}
 	}
+	return err;
+}
+
+error::Error DoAuthenticate(context::MenderContext &main_context) {
+	events::EventLoop loop;
+	auto &config = main_context.GetConfig();
+	if (config.server_url == "") {
```

Comment:
Related to [this](https://github.com/mendersoftware/mender/pull/1408/files#r1341056123), shouldn't you also check here for entries in the servers array?

Reply:
Depends who merges first? :laughing: 

Alright, but probably safest to change it.. or not? :thinking: 

Honestly not sure. You decide

Reply:
Leave it as is.

(and merge fast!)